missions/ULM_missions.txt

```diff
@@ -0,0 +1,335 @@
+ULM_diplomacy = {
+	slot = 1
+	generic = no
+	ai = yes
+	potential = {
+		tag = ULM
+	}
+	has_country_shield = yes
+
+
+	ULM_hre_allies = {
+		icon = mission_alliances.png
+		required_missions = { }
+		position = 1
+		
+		trigger = {
+			calc_true_if = { # alejandro = god
+				amount = 2
+				all_countries = {
+					is_part_of_hre = yes
+					allied_to = ROOT
+				}
+			}
+		}
+
+		effect = {
+			prestige = 20
+			add_country_modifier = {
+				name = "regional_diplomacy"
+				duration = 9125 #25 years
+			}
+		}
+	}
+	
+	ULM_imperial_guarantee = {
+	icon = mission_monarch_in_throne_room.png
+	required_missions = { ULM_hre_allies }
+	position = 2
+		
+		trigger = {
+			calc_true_if = { # alejandro = god
+				amount = 1
+				OR = {
+					all_countries = {
+						is_emperor = yes
+						allied_to = ROOT
+					}
+					all_countries = {
+						is_emperor = yes
+						has_opinion = {
+							who = ROOT
+							value = 100
+						}
+						owes_favors = { who = ULM value = 10 }
+					}
+				}
+			}
+		}
+
+		effect = { # replace with event
+			add_dip_power = 100
+			add_prestige = 20
+			add_imperial_influence = 25
+			all_countries = {
```

Comment:
Any reason for `all_countries` instead of `emperor = { }`?

Reply:
I'm not good at coding syntax yet

Reply:
Unresolving. Not replaced.